.github/workflows/ci.yml

```diff
@@ -132,7 +132,7 @@ jobs:
         run: sbt "docs/mdoc"
 
       - name: Build Microsite
-        run: cd ./modules/website && yarn && yarn build
+        run: cd ./modules/website && yarn && yarn swizzle docusaurus-lunr-search SearchBar --danger && yarn build
```

Comment:
good, I saw this last night and I was going to ask if we could use yarn everywhere, happy it was addressed